nerfstudio/process_data/colmap_utils.py

```diff
@@ -418,6 +418,14 @@ def colmap_to_json(
     # im_id_to_image = recon.images
     cam_id_to_camera = read_cameras_binary(recon_dir / "cameras.bin")
     im_id_to_image = read_images_binary(recon_dir / "images.bin")
+    multifocal = False  # one camera for all frames (distort_fixed=True)
+    if set(cam_id_to_camera.keys()) != {1}:
+        print("Only single camera shared for all images is supported.")
```

Comment:
Given the change, I think now we are parsing multiple cameras from colmap. So just give a warning message indicating that more than one camera is found in the colmap project should suffice. We can set `out={}` here.